dags/benchmark/pytorchxla_torchbench.py

```diff
@@ -19,31 +19,111 @@
 from configs import composer_env, vm_resource
 from configs.benchmark.pytorch import pytorchxla_torchbench_config as config
 
-_MODELS = [
-    "BERT_pytorch",
-    "Background_Matting",
-    "hf_Bert",
-    "Super_SloMo",
-    "tts_angular",
-]
+# Schudule the job to run once per two days at 5:00PM UTC.
+SCHEDULED_TIME = "0 17 */2 * *" if composer_env.is_prod_env() else None
+NETWORK_PREFIX = "projects/tpu-prod-env-automated"
+V5_NETWORKS = f"{NETWORK_PREFIX}/global/networks/mas-test"
+V5E_SUBNETWORKS = f"{NETWORK_PREFIX}/regions/us-east1/subnetworks/mas-test"
+V5P_SUBNETWORKS = f"{NETWORK_PREFIX}/regions/us-east5/subnetworks/mas-test"
 
-# Schudule the job to run every day at 5:00PM UTC.
-SCHEDULED_TIME = "0 17 * * *" if composer_env.is_prod_env() else None
 
 with models.DAG(
     dag_id="pytorchxla-torchbench",
     schedule=SCHEDULED_TIME,
     tags=["pytorchxla", "nightly", "torchbench"],
-    start_date=datetime.datetime(2023, 8, 29),
+    start_date=datetime.datetime(2024, 1, 1),
```

Comment:
nit question: why is the date hardcoded?

Reply:
This start date does not quite matter, if we set up [catchup](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dag-run.html#catchup) as False (so no backfill test will run). It indicates when we initialize the scheduled tests.